scipy/stats/_pagel.py

```diff
@@ -0,0 +1,287 @@
+
+    However, this standardized statistic does not distinguish between the
+    observed values being well correlated with the predicted ranks and being
+    _anti_-correlated with the predicted ranks. Instead, we follow [2]_
+    and calculate the standardized statistic
+
+    .. math::
+
+        \Lambda = \frac{L - E_0}{\sqrt{V_0}},
+
+    where :math:`E_0 = \frac{1}{4} mn(n+1)^2` and
+    :math:`V_0 = \frac{1}{144} mn^2(n+1)(n^2-1)`, "which is asymptotically
+    normal under the null hypothesis". The asymptotic calculation is not
+    adjusted for the possibility of ties, which, according to [2]_, does not
+    violate the significance level of the result.
+
+    The *p*-value for ``method='exact'`` is generated by comparing the observed
+    value of :math:`L` against the :math:`L` values generated for all
+    :math:`(n!)^m` possible permutations of ranks. For ``method='mc'``,
+    the observed value of :math:`L`  is compared against ``n_s``
+    randomly sampled rank permutations. If the calculation is to be adjusted
+    for the possibility of ties, then the observed :math:`L` is compared
+    against the :math:`L` values generated for all  :math:`(m^m)^n` possible
+    tables of ranks (`exact`) or a randomly generated subset of them (`mc`).
+
+
+    References
+    ----------
+    .. [1] Ellis Batten Page, "Ordered hypotheses for multiple treatments:
+       a significant test for linear ranks", *Journal of the American
+       Statistical Association* 58(301), p. 216--230, 1963.
+
+    .. [2] Markus Neuhauser, *Nonparametric Statistical Test: A computational
+       approach*, CRC Press, p. 150--152, 2012.
+
+    .. [3] Statext LLC, "Page's L Trend Test - Easy Statistics", *Statext -
+       Statistics Study*, https://www.statext.com/practice/PageTrendTest03.php,
+       Accessed July 12, 2020.
+
+    Examples
+    --------
+    We use the example from [3]_: 10 students are asked to rate three
+    teaching methods - tutorial, lecture, and seminar - on a scale of 1-5,
+    with 1 being the lowest and 5 being the highest. We have decided that
+    a confidence level of 99% is required to reject the null hypothsis in favor
+    of our alternative: that the seminar will have the highest ratings and the
+    tutorial will have the lowest. Initially, the data have been tabulated with
+    each row representing an individual student's ratings of the three methods
+    in the following order: tutorial, lecture, seminar.
+
+    >>> table = [[3, 4, 3],
+                 [2, 2, 4],
+                 [3, 3, 5],
+                 [1, 3, 2],
+                 [2, 3, 2],
+                 [2, 4, 5],
+                 [1, 2, 4],
+                 [3, 4, 4],
+                 [2, 4, 5],
+                 [1, 3, 4]]
+
+    We rank the data such that high scores correspond with high ranks, settling
+    ties with an average rank:
+
+    >>> from scipy.stats import rankdata
+    >>> ranks = rankdata(table, axis=1)
+    >>> print(ranks)
```

Comment:
minor: better not to use `print()` at all. I know lots of old `stats` examples still have that, but let's teach users to just let IPython evaluate the variable.

Reply:
OK I wasn' sure whether this would be picked up by the doctest.

Reply:
I implemented this suggestion, but since then I realized that I appreciate it when the examples include `print` explicitly because it can be copied and pasted into a script, not just the console. I also think it's nice when they `import numpy as np`. It's not ideal when bugs are reported with code that doesn't "just work" - shouldn't our docstrings follow the same standard?

Reply:
I see what you mean, but we've had the convention of intentionally not including `import numpy as np` for a very long time, and it seems to have worked out OK.  If you are interested in changing that, the mailing list is the place to start that discussion.

I also lean towards replacing `>>> print(ranks)` with just `>>> ranks`.  Again, that's the convention we've been using for a long time.   The only case where I would use the print function is when I specifically want to show the `__str__` output instead of the `__repr__` output.  I don't think we need that for a plain numpy array.  (For example, I'll use print in the new ANOVA functions' docstrings when they eventually show up in a PR, because the `__repr__` will be the standard `dataclass`-like output, but the `__str__` method will generate the classic ANOVA summary table.)

Reply:
I had already removed all the `prints`, so I think this is resolved. There are bigger fish to fry!